packages/api-explorer/src/Params.jsx

```diff
@@ -133,6 +131,7 @@ class Params extends React.Component {
 Params.propTypes = {
   ArrayField: PropTypes.func.isRequired,
   BaseInput: PropTypes.func.isRequired,
+  DateTimeWidget: PropTypes.func.isRequired,
```

Comment:
what does [prop-types](https://www.npmjs.com/package/prop-types) do? it says it checks for React props, but the [usage section](https://www.npmjs.com/package/prop-types#usage) doesn’t seem too clear on what it’s actually doing. in the [React Docs](https://reactjs.org/docs/typechecking-with-proptypes.html), it looks like it’s a kind of validator that makes sure the props that are being passed are of correct type?

Reply:
Yep that's what it does. Ensures that when you pass one of these properties to a React component that it's of the type specified here. If it isn't, React will throw a soft error in the console.

Reply:
oops. accidentally messed up the review requesters 😓